src/libraries/System.Text.Json/gen/JsonSourceGenerator.Emitter.cs

```diff
@@ -152,7 +152,8 @@ private void AddSource(string fileName, string source, bool isRootContextDef = f
                 bool isInGlobalNamespace = @namespace == JsonConstants.GlobalNamespaceValue;
 
                 StringBuilder sb = new(@"// <auto-generated/>
-#nullable enable
+#nullable enable annotations
```

Comment:
Maybe add an empty line between this and the top declaration?
```suggestion

#nullable enable annotations
```